src/data/real/ennreal.lean

```diff
@@ -1057,6 +1058,18 @@ le_inv_iff_le_inv.trans $ by rw inv_one
 @[simp] lemma inv_lt_one : a⁻¹ < 1 ↔ 1 < a :=
 inv_lt_iff_inv_lt.trans $ by rw [inv_one]
 
+/-- The inverse map `λ x, x⁻¹` is an order isomorphism between `ℝ≥0∞` and its `order_dual` -/
+@[simps apply]
+def order_iso_inv_dual : ℝ≥0∞ ≃o order_dual ℝ≥0∞ :=
```

Comment:
I think something like `inv_order_iso_dual` would be fit the naming scheme better, in line with things like `star_ring_aut`.

Reply:
I'm fine with making this change, but it seems there is not widespread agreement about this, e.g., when I instantiated `rpow` as an order isomorphism I had it originally as `rpow_order_iso` but someone asked me to switch it to `order_iso_rpow`.

Reply:
Asked on Zulip: https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/naming.20defs